bin/eslint.js

```diff
@@ -8,10 +8,11 @@ process.chdir(CODE_DIR);
 var stdout = console.log;
 console.log = console.error;
 
-var eslint = require('../lib/eslint-patch')(require('eslint'));
+
+var eslint = require('../lib/eslint-patch')();
 
 var CLIEngine = eslint.CLIEngine;
-var docs = eslint.docs;
```

Comment:
Cool. We use the docs for the issue contents - as long as those still work we're probably ok